lib/ex_doc/markdown.ex

```diff
@@ -8,9 +8,9 @@ defmodule ExDoc.Markdown do
   * [Earmark][]
   * [Pandoc][]
 
-  If you don't offer any preference via `config/config.exs`. ExDoc will try to
-  find one of the earlier MarkDown parsers. Otherwise, ExDoc will raise an
-  exception.
+  If you don't specify a parser in `config/config.exs`. ExDoc will try to
+  find one of the MarkDown parser from the list in top down fashion. Otherwise, 
```

Comment:
Btw, Markdown is spelled "Markdown" and not "MarkDown". I know this was spelled wrong before, just pointed out in case you want to fix this in another PR. Thank you! :)